pages/index.tsx

```diff
@@ -60,16 +69,21 @@ export default function Home({ onLike, likes }: HomeProps): JSX.Element {
 
       {/* Switching between the three different views */}
 
-      {viewPoint.random && <RandomView />}
+      {viewPoint.random && <RandomView artists={artists} />}
       {viewPoint.artists &&
         artists.map((artist) => (
-          <List key={artist.id} {...artist} onLike={onLike} likes={likes} />
+          <List key={artist._id} {...artist} onLike={onLike} likes={likes} />
         ))}
       {viewPoint.favorites &&
         artists.map(
           (artist) =>
-            likes.includes(artist.id) && (
-              <List key={artist.id} {...artist} onLike={onLike} likes={likes} />
+            likes.includes(artist._id) && (
+              <List
+                key={artist._id}
+                {...artist}
+                onLike={onLike}
+                likes={likes}
+              />
```

Comment:
You could filter first for the liked artists and map on these:

```suggestion
            artists.filter(artist => likes.includes(artist._id))
            artists.map(artist => (
              <List
                key={artist._id}
                {...artist}
                onLike={onLike}
                isLiked
              />
           ))  
```
As mentioned in my last comment, you would then pass the `isLiked` property for each favourite artist.